### DISCUSSION TITLE
On 1809 1.2.1 glass doesn't extend anywhere

### DISCUSSION THREAD

**Discussion by ideapc** [ALTaleX531/OpenGlass#48] [Q&A] [answered]
On my end StartIsBack taskbar+start menu, settings app (and other metro applets) and windows borders all stay untouched and have no glass, while it works for the titlebar.
Any help?

**Comment by ideapc** [comment]
I managed to fix borders by setting GlassOverrideBorder to 1, now it extends to the border of window.
But still no luck with extending the blur to the other items, especially the StartIsBack ones (taskbar and start menu).

**Comment by ALTaleX531** [comment]
If you try a little harder, you can find a useful option called `GlassOverrideAccent`.

**Reply by ideapc** [reply]
What do I set it to? For me its already 1

**Reply by ideapc** [reply]
It actually works for startallback start menu but not taskbar. Taskbar stands transparent, though shadowed.

**Reply by ALTaleX531** [reply, marked as answer]
I think it's a startisback bug, startisback 2.9.20 doesn't work properly in 1809, startisback can't adjust the transparency of the taskbar even if openglass is not enabled. openglass adjusts the transparency of the taskbar depending on the actual transparency of the taskbar.

**Reply by ideapc** [reply]
Thank you so much. Makes sense then.
